src/azure-cli/azure/cli/command_modules/backup/custom_help.py

```diff
@@ -521,6 +521,11 @@ def get_resource_group_from_id(arm_id):
     return m.group(0)
 
 
+def get_subscription_from_id(arm_id):
+    m = re.search('(?<=/subscriptions/)[^/]+', arm_id)
```

Comment:
Please add comments or examples for the regular expressions

Reply:
Added comments to explain the regex